crates/burn-import/src/burn/node/slice.rs

```diff
@@ -0,0 +1,104 @@
+use super::{Node, NodeCodegen};
+use crate::burn::{Scope, TensorType, Type};
+use burn::record::PrecisionSettings;
+use proc_macro2::TokenStream;
+use quote::quote;
+
+#[derive(Debug, Clone, new)]
+pub struct SliceNode {
+    pub input: TensorType,
+    pub output: TensorType,
+    pub starts: Vec<usize>,
+    pub ends: Vec<usize>,
+    // pub axes: Option<Vec<i64>>,
+    // pub steps: Option<Vec<i64>>,
+}
+
+impl<PS: PrecisionSettings> NodeCodegen<PS> for SliceNode {
+    fn output_types(&self) -> Vec<Type> {
+        vec![Type::Tensor(self.output.clone())]
+    }
+    fn input_types(&self) -> Vec<Type> {
+        vec![Type::Tensor(self.input.clone())]
+    }
+    fn forward(&self, scope: &mut Scope, node_position: usize) -> TokenStream {
+        let input = scope.tensor_use_owned(&self.input, node_position);
+        let output = &self.output.name;
+        // let axes = match &self.axes {
+        //     Some(axes) => axes.to_tokens(),
+        //     None => quote! { None },
+        // };
+        // let steps = match &self.steps {
+        //     Some(steps) => steps.to_tokens(),
+        //     None => quote! { None },
+        // };
+
+        // compine starts and ends into ranges
+        // example: starts = [0, 0, 0, 0], ends = [1, 1, 1, 1]
+        // ranges = [0..1, 0..1, 0..1, 0..1]
+        let starts = &self.starts;
+        let ends = &self.ends;
```

Comment:
I think we still need to check the axes since there is no guarantee that they always including the leading axes (e.g., if `axes = [2, 3]` for values of `starts` and `ends`), but it's not entirely clear even as described in the [spec](https://github.com/onnx/onnx/blob/main/docs/Operators.md#Slice).

If we keep this solution without checking for non leading axes, I would at least parse the values of `axes` and error on invalid/unsupported values (i.e., skipping leading axes).